src/spec/onPrepare.js

```diff
@@ -1,3 +1,4 @@
+browser.driver.manage().window().setSize(1280, 960);

```

Comment:
Could you add a `// TODO` mark to indicate that it is missing to set _saucelabs_ display size?